src/README.md

```diff
@@ -99,6 +99,18 @@ Tables of named input and output links.  These tables are initialized by
 the engine for use in processing and are *read-only*.
 
 
+— Field **myapp.appname**
+
+Name of the app. *Read-only*.
+
+
+— Method **myapp:start**
+
+*Optional*. Start the app. At this the `input` and `output` link tables are
+populated, and `appname` is set. The app can perform additional initialization
+if required.
```

Comment:
@lukego @dpino What do you think about the name `configured`, e.g. the callback to be called after an app was configured?